cbdcconfig/defaults.go

```diff
@@ -1,9 +1,11 @@
 package cbdcconfig
 
 const (
-	DEFAULT_AWS_REGION       = "us-west-2"
-	DEFAULT_AZURE_REGION     = "westus2"
-	DEFAULT_GCP_REGION       = "us-west1"
-	DEFAULT_CAPELLA_ENDPOINT = "https://api.cloud.couchbase.com"
-	DEFAULT_CAPELLA_PROVIDER = "aws"
+	DEFAULT_AWS_REGION        = "us-west-2"
+	DEFAULT_AZURE_REGION      = "westus2"
+	DEFAULT_GCP_REGION        = "us-west1"
+	DEFAULT_CAPELLA_ENDPOINT  = "https://api.cloud.couchbase.com"
+	DEFAULT_CAPELLA_PROVIDER  = "aws"
+	DEFAULT_SEVEN_SIX_VERSION = "7.6.0"
```

Comment:
What is this for?  We shouldn't have anything specific to any server version here, as it makes it harder in the future to support later releases.

Reply:
In /deploy path in body server version should be passed like 7.6.0 or 7.6.1 but in other one it should be like 7.6, 7.2, otherwise it gives error message server version is not supported. So for path /deploy i am checking if 7.6 is substring of 7.6.0, then updating it to 7.6.0

Reply:
@brett19 I have remove handling this case, as it is not needed and i think cbdinoclients should be handling this